indico/modules/events/contributions/templates/display/subcontribution_display.html

```diff
@@ -0,0 +1,118 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% from 'attachments/_attachments.html' import render_attachments %}
+
+{% block title -%}
+    {{ subcontrib.contribution.title }}
+{%- endblock %}
+
+{% block subtitle -%}
+    {{ subcontrib.title }}
+{%- endblock %}
+
+{% block actions -%}
+    <div class="toolbar right">
+        {% if subcontrib.can_manage(session.user) -%}
+            <a href="#" class="i-button icon-edit"
+               data-title="{% trans title=subcontrib.title %}Edit subcontribution '{{ title }}'{% endtrans %}"
+               data-href="{{ url_for('.manage_edit_subcontrib', subcontrib) }}"
+               data-ajax-dialog
+               data-reload-after></a>
+        {%- endif %}
+    </div>
+{%- endblock %}
+
+{% block content -%}
+    <div id="subcontribution-display">
+        <div class="infogrid condensed">
+            <div class="infoline">
+                <i class="icon-time"></i>
+                <div class="text">
+                    <div>
+                        <span>
+                            {{ subcontrib.duration|format_human_timedelta }}
+                        </span>
+                    </div>
+                </div>
+            </div>
+            {%- set location_data = subcontrib.location_parent.location_data -%}
+            {% if location_data.venue  -%}
+                <div class="infoline">
+                    <i class="icon-location"></i>
+                    <div class="text">
+                        <div>
+                            <span>
+                                {# TODO: show the room on the map (proper redirect) #}
```

Comment:
Why don't we use [what I've made for meetings](https://github.com/indico/indico/blob/devel/indico/modules/events/templates/display/indico/_common.html#L28)?